diff --git a/lib/transpiler/transpiler.js b/lib/transpiler/transpiler.js
index 3c26e5852787f0246065f6206ba0901ee32b16a7..370266a29f2c618ae48c50f4acc7462ac6cc0b4f 100644
--- a/lib/transpiler/transpiler.js
+++ b/lib/transpiler/transpiler.js
@@ -90,7 +90,8 @@ function transpileFiles(directory, outputDir, options) {
                             dir: outputDir,
                             exports: "auto",
                             paths: {
-                                'react/jsx-runtime': 'react/cjs/react-jsx-runtime.production.min',
+                                // Make sure it finds the correct version of react https://github.com/asyncapi/generator-react-sdk/pull/242
+                                'react/jsx-runtime': require.resolve('react/cjs/react-jsx-runtime.production.min'),
                             },
                             sanitizeFileName: false,
                         })];
